Dockerfile

```diff
@@ -18,11 +20,11 @@ RUN npm prune --production
 
 FROM node:16-alpine
 
-RUN apk add --no-cache curl docker-cli grep
+RUN apk add --no-cache curl docker-cli docker-compose grep
 
```

Comment:
RUN apk update && apk add --no-cache yarn curl bash tini git docker-cli docker-compose grep make python3 g++

RUN apk add --no-cache curl docker-cli docker-compose grep

why do dependancies have to be installed multiple times?

Reply:
The first layer is only for building the package, there has no relationship for the dependencies cross different layers